### DISCUSSION TITLE
Launch from command line?

### DISCUSSION THREAD

**Discussion by kevinashworth** [lukehaas/RunJS#594] [Q&A] [answered]
Is there a way to open RunJS from the command line? And in particular a way to launch it with a certain file from the command line?

With Visual Studio Code, I can type `code file.js` to open `file.js` from the command line. So something like, `runjs file.js` is what I'm looking for.

(I'm on macOS.)

**Comment by lukehaas** [comment]
@kevinashworth currently not but I will consider adding this option in a future release 👍

**Comment by wsyeung** [comment, marked as answer]
Before Luke adds the feature (or already as I am testing with 4.1.0), @kevinashworth you may try following which works for me (macOS 26.5.2 M1 Pro CPU).
`open -a runJS file.js`
